src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -449,17 +447,6 @@ const MarketplaceRow = ({
       <Row style={{ flex: '1.5' }} display={{ sm: 'none', md: 'flex' }}>
         <Column width="full">
```

Comment:
nit: this Column can be removed since profit has been removed. I'd just check to see if the width style potentially needs to get added to the Row above